build/cross-compile.sh

```diff
@@ -76,11 +72,11 @@ case "$SUBCOMMAND" in
         shift
         ;;
     *)
-        echo "Undefined architecture for cross-compile. Supported architectures: linux-386, linux-amd64, linux-arm-5, linux-arm-6, linux-arm-7, linux-arm64, linux-mips, linux-mipsle, linux-mips64, linux-mips64le, darwin-386, darwin-amd64, windows-386, windows-amd64"
+        echo "Undefined architecture for cross-compile. Supported architectures: linux-386, linux-amd64, linux-arm-5, linux-arm-6, linux-arm-7, linux-arm64, linux-mips, linux-mipsle, linux-mips64, linux-mips64le, darwin-amd64, windows-386, windows-amd64"
         echo "Usage: ${0} {arch} [kcn|kpn...]"
-        echo "    ${0} linux-386"
-        echo "    ${0} darwin-386 kcn"
-        echo "    ${0} windows-386 kcn kpn"
+        echo "    ${0} linux-amd64"
```

Comment:
Do we also need to change `linux-386` to `linux-amd64`? (Also for windows-386)

Reply:
Not necessary. I thought aligning them looks better and most user use 64bit than 32bit. @ehnuje Do you think changing only darwin-32 is better?

Reply:
No I was just curious. Thanks for the explanation!